src/client/components/homeComponents/HomePosts.jsx

```diff
@@ -46,6 +48,10 @@ const HomePosts = (props) => {
     getPosts();
   }, []);
 
+  const handleClick = () => {
+    console.log("Theunis");
+  }
+
```

Comment:
I'm guessing this is here for testing, best to remove this before merging the PR